ext/pxp-agent.logrotate

```diff
@@ -0,0 +1,11 @@
+/var/log/puppetlabs/pxp-agent/*.log {
+    daily
```

Comment:
Probably depends on what kind of info is in the log? Daily & keep 30 works for me for now? Or weekly, keep 4-ish?